### ISSUE TITLE
On Elementary, put Desktop Folder in the switchboard-plug-applications startup tab.

### ISSUE THREAD

**Issue by eljefuri** [spheras/desktopfolder#143]
To make it clear to the user how to make Desktop Folder start and stop upon login, why not include it in Elementary's [switchboard-plug-applications](https://github.com/elementary/switchboard-plug-applications) "Startup" tab, instead of creating a startup script that is for the most part hidden to novice users?

**Comment by fossfreedom** [comment]
Can I just say... all excellent suggestions for all the issues you raised today. 

I am sure any help in actually implementing any of these would help the project immensely.

**Comment by spheras** [comment]
Yes, I agree that could be great for elementary users. I'm not sure how to implement that, and if doing that I could break the generic compatibility with other systems, probably not... but need to be investigated.  [Elementary Tweaks project](https://github.com/elementary-tweaks/elementary-tweaks) could be an example of that.

Regarding @fossfreedom  comment, I agree, it is a lot of issues/work suddenly :smile: In general, the issues are great, the problem is just to find the equilibrium between useful and difficult.  A bounty system could help, maybe, I'm not experienced in that systems, to be honest, I'm a little afraid to introduce the variable money into the project.

**Comment by eljefuri** [comment]
@spheras Honestly I am not a big believer in bounties and I am not sure that they _directly_ contribute much to most projects. Probably the only point of introducing them is to make certain issues stand out amongst all the others. As a signal e.g. `someone cared enough about this topic to bother putting up a bit of cash`, a few things are subliminally understood:

- There are _real people_ who care about the fate of the app.
- Said real people must either really be using it, or desperately wanting to.
- There is an opportuntiy for an act of heroism.

**Comment by fossfreedom** [comment]
However the request is handled - and whoever does it - FYI - if you poke gsettings key com.github.spheras.desktopfolder  show-desktopfolder to be "false" then desktopfolder will not start on logon / quit if started.  

i.e. you no longer have to delete the autostart .desktop file.

**Comment by spheras** [comment]
@megatux, what I don't understand is why you need that behavior in elementary. I mean, in elementary this is an external app, and you install it if you want. At the moment you install it, it is suposed you want to have it running in your desktop... so, if you don't want it, just remove the application from the system. Why do you want the application if it is not running?? Do you launch it manually?? :D That's what I don't see.

**Comment by megatux** [comment]
So, how it is currently working? If I want to use the app and installed it, will it start also on the rest of users created in the system on next login? Normally if you want a program to always start on log in, you configure it explicitly for you, in the desktop settings or on an app setting, but that's something that only affects your custom config.

**Comment by spheras** [comment]
@megatux, so the problem is that it affects to all the users, isn't it? mmm interesting

**Comment by eljefuri** [comment]
For me the problem is that Elementary provides a user-level preference pane (switchboard-plug-applications) for applications to auto-launch, and for consistency's sake I think it should be used. But @megatux also has a great point that I had not considered.

**Comment by nnmlss** [comment]
I came here because I have uninstalled the application from my elementary OS long time ago but the autostart script still resides and launches nothing on every startup. If I  remove the .desktop file from ~/.config/autostarts soon I see it again there.. It is annoying, so here I found the answer, I have to remove the /etc/xdg/autostart/... file. I think it will be more clean if the application removes it alone if you uninstall it.
